enhancements/sig-policy/98-long-term-compliance-history/README.md

```diff
@@ -332,6 +336,11 @@ was evaluated. With this additional information, the Status Sync controller can
 `/api/v1/compliance-events` `POST` request. If the spec couldn't be determined, a value of `unknown` will be recorded.
 The rest of the data can be found in the Kuberenetes `Event` object.
 
+To record a compliance event when a policy is deleted, the Spec Sync controller will record these compliance events
```

Comment:
Should it basically work the same way if the policy is disabled?

Reply:
Right, because when a policy is disabled on the hub, it deletes the propagated policy which causes the policy templates to be deleted on the cluster too.

Reply:
@gparvin , I think the timestamp and an adjustment to the message would suffice for now.

Since our main focus is currently on OCM policies, I think we are covered. We may need to adjust if we pursue the scraping idea.